test/airport/_mocks/airportJsonMock.js

```diff
@@ -1,532 +1,532 @@
 /* eslint-disable */
 export const AIRPORT_JSON_KLAS_MOCK = {
-  "radio": {
```

Comment:
Forgot I left that in there (but there's so many changed lines it didn't even catch my attention!). It was an auto-indent on the mock, so it all folds/unfolds properly (I do more code folding than you would ever believe, and the inability was driving me nuts).